### DISCUSSION TITLE
Clearable q-input date picker should reverse order buttons for clearing and date picking

### DISCUSSION THREAD

**Discussion by MichaelMarkieta** [quasarframework/quasar#10026] [Ideas / Proposals]
I would have expected the clearable button to be placed on the left of the calendar button, to match other q-input UX / UI.

![image](https://user-images.githubusercontent.com/3111752/125153375-3b4cee80-e121-11eb-87c3-c6186b292603.png)

**Comment by ontwikkelfabriek** [comment]
@MichaelMarkieta

You could be right. What does the Material Design Spec say about this?
